fgd/point/point/point_posecontroller.fgd

```diff
@@ -35,4 +35,5 @@
 	input SetFModRate(float) : "Set the modulation cycles per second from -10.0f to 10.0."
 	input SetFModAmplitude(float) : "Set the modulation extents from 0.0f to 10.0."
 	input RandomizeFMod(float) : "Randomize the frequency modulation by an extremeness of 0.0 to 1.0."
+	input GetFmod(void) : "Outputs the current modulation settings in console."
```

Comment:
All this does is print a dev-only message to console - I think it's fine to keep out of the FGD.

Reply:
Unless something is clearly broken, or unusable in said game is there much of a reason to keep it out?